pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -805,19 +805,19 @@ public static void deactivateKitRequest(long dsmKitRequestId, @NonNull String de
         }
     }
 
-    public static KitRequestShipping getKitRequest(@NonNull String kitRequestId) {
+    public static KitRequestShipping getKitRequest(String dsmKitRequestId) {
```

Comment:
While we're changing this, can we change `dmsKitRequestId` to an int (along with setInt on L812?  It's type in the database is int.